### PR TITLE
Browser: fix control panel crash

### PR DESCRIPTION
**Pull request by christianbaroni**
## What changed (plus any additional context for devs)
- Fixes a crash that occurred when opening the control panel after creating/importing a wallet from a fresh install

### CODE REVIEW

File: src/utils/connectedApps.ts [hunk 1]
```diff
@@ -9,24 +9,32 @@ export const isValidUrl = (url: string) => {
...
   if (!url) return '';
-  const host = new URL(url).host;
-  if (host.indexOf('www.') === 0) {
-    return host.replace('www.', '');
+  try {
+    const host = new URL(url).host;
+    if (host.indexOf('www.') === 0) {
+      return host.replace('www.', '');
+    }
+    return host;
```
Comment (walmat) [inline src/utils/connectedApps.ts:17]: small nitpick, but the check is redundant and unnecessary. Instead just call `.replace` on it no matter what
```suggestion
    const host = new URL(url).host;
    return host.replace('www.', '');
```

File: src/utils/connectedApps.ts [hunk 2]
```diff
@@ -9,24 +9,32 @@ export const isValidUrl = (url: string) => {
...
+    const subdomains = urlObject.hostname.split('.');
+    if (subdomains.length === 2) {
+      hostname = urlObject.hostname;
+    } else {
+      hostname = `${subdomains[subdomains.length - 2]}.${subdomains[subdomains.length - 1]}`;
+    }
+    return hostname;
+  } catch (err) {
+    return '';
   }
```
Comment (walmat) [inline src/utils/connectedApps.ts:37]: what is `subdomains length` is `0` here?

Comment (christianbaroni) [reply src/utils/connectedApps.ts]: I believe a valid URL will always have at least 1 part (e.g. localhost), and usually 2+ parts (domain.com)

I pushed a fix to handle the 1-part URL edge case — I think everything else is handled by the catch

File: src/utils/connectedApps.ts (hunk 1 above)
Comment (christianbaroni) [reply src/utils/connectedApps.ts:17]: I think technically www can be a subdomain that isn't necessarily at the start of the URL

**Review by ibrahimtaveras00 — APPROVED** [review]
🌮

**Review by brunobar79 — APPROVED** [review]
🚀

**Pull request merged**